spec/features/user_email_is_valid_on_sign_up_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.feature "Sign-up email", type: :feature do
+  scenario "User email has correct format" do
+    visit "/users/sign_up"
```

Comment:
The web helper method sign_up might be useful here, with an email and password as arguments.